usr.sbin/unbound/setup/local-unbound-setup.sh

```diff
@@ -195,7 +195,9 @@ do_not_edit() {
 gen_resolvconf_conf() {
 	local style="$1"
 	do_not_edit
-	echo "resolv_conf=\"/dev/null\" # prevent updating ${resolv_conf}"
+  echo "name_servers_append=\"127.0.0.1\""
+  echo "resolv_conf_local_only=\"yes\""
+  echo "resolv_conf_options=\"edns0\""
```

Comment:
058a4e3419425 removed this because:

    Prevent resolvconf from updating /etc/resolv.conf.  As Jakob Schlyter
    pointed out, having additional nameservers listed in /etc/resolv.conf
    can break DNSSEC verification by providing a false positive if unbound
    returns SERVFAIL due to an invalid signature.  The downside is that
    the domain / search path won't get updated either, but we can live
    with that.
    
    Approved by:    re (blanket)

Reply:
There won't be additional nameservers, `resolv_conf_local_only` makes resolvconf only add `127.0.0.1` and it's also smart enough not to add the localhost to `forward.conf`.

Reply:
> There won't be additional nameservers, `resolv_conf_local_only` makes resolvconf only add `127.0.0.1` and it's also smart enough not to add the localhost to `forward.conf`. Resulting `resolv.conf` is similar to what the script was generating except commented out old records.
> 
> ```
> petruchito@delta:~ % cat -n /etc/resolv.conf
>      1	# Generated by resolvconf
>      2	nameserver 127.0.0.1
>      3	options edns0
>      4	
> ```

In other words this reverts 058a4e3 because 058a4e3 removed the add of nameserver 127.0.0.1. Correct?

Reply:
Yes. 
I just thought I should also have added `echo "resolv_conf=\"${resolv_conf}\""` for non-standard `resolv.conf` locations.